src/cases/goodluck/add_remove.js

```diff
@@ -8,25 +8,37 @@ class World extends WorldImpl {
 const HAS_A = 1 << 0;
 const HAS_B = 1 << 1;
```

Comment:
Both the `HAS_*` masks and the mixin thunks could actually be static methods on the component classes, but I'm not sure if it's a good idea to drift away so much from upstream Goodluck. What do you think, @ooflorent?

Reply:
Since you’re one of the maintainer of goodluck I think you know better than me!

Reply:
I'm going to keep it as-is. Goodluck is all about extreme simplicity after all.